java-checks-test-sources/src/main/java/checks/MutableMembersUsageCheck.java

```diff
@@ -26,6 +33,93 @@ class MutableMembersUsageCheck {
   private List<String> customUnmodifiableList1 = customUnmodifiableList(mutableList);
   private List<String> customUnmodifiableList2 = CustomUnmodifiableList.create(mutableList);
 
+  private Map<String, String> hashMap = new HashMap<>();
+
+  public Map<String, String> getHashMap() {
+    return hashMap; // Noncompliant
+  }
+
+  private ImmutableMap<String, String> immutableMap;
+
+  public Map<String, String> getImmutableMap() {
+    return immutableMap; // Compliant
+  }
+
+  private Map<String, String> immutableMap2 = ImmutableMap.of("abc", "abc");
+
+  public Map<String, String> getImmutableMap2() {
+    return immutableMap2; // Compliant
+  }
+
+  private static final Set<String> unmodifiableFromStream = // FP
```

Comment:
Are this and the following field still FPs?

Reply:
Oh, probably leftovers. Will remove